app/styles/variables.css

```diff
@@ -85,6 +85,7 @@
   --color-green-8: #108944;
   --color-green-9: #06371b;
 
+  --color-red-hightlight: #ffeaea;
```

Comment:
Did you try out the `var( --color-red-hover-alpha)` variable? I feel like its purpose is the same thing, so it would be nice if it matched the other stuff, e.g. when you hover joblistings on _/karriere_). You can choose what looks best though.

Reply:
I replaced it with the --color-red-hover-alpha, although from a universal design perspective I think the hover effect should be less transparent so it's more visible. We can look at it later though:)